types/types.go

```diff
@@ -121,6 +121,7 @@ const (
 // In some situations, some fields may be unknown, in others they may be mandatory; documenting an “unknown” value here does not override that.
 type BlobInfo struct {
 	Digest      digest.Digest // "" if unknown.
+	TOCDigest   digest.Digest // "" if unknown.
```

Comment:
From a quick check, I don’t see any users; but we _do_ have locations (like `TryReusingBlobWithOptions`) where `Digest` is documented to be required… so I at a first glance I don’t see that the way `tryReusingBlobAsPending` is setting only `TOCDigest` is sufficient.

That would be some kind of large-scale overhaul… similarly to `c/storage.Layer.UncompressedDigest`, but probably worse.

Reply:
dropped